### PR TITLE
Transform: Adds scheduler component

### PR DESCRIPTION
**Pull request by gildub**
This complement the master configuration with Project Configuration DefaultNodeSelector which is part of scheduler in OCP4

### CODE REVIEW

**Comment by arapov** [comment]
@gildub this patch looks good, though it wants to have tests fixed.

File: pkg/utils/test/load_test_data.go [hunk 1]
```diff
@@ -11,6 +11,23 @@ import (
 	"k8s.io/client-go/kubernetes/scheme"
 )
 
+// LoadMasterConfig loads MasterConfig
+func LoadMasterConfig(file string) (*legacyconfigv1.MasterConfig, error) {
```
Comment (arapov) [inline pkg/utils/test/load_test_data.go]: @Danil-Grigorev has pointed to this piece. It looks OK to do, but @gildub is there other places to use LoadMasterConfig in the code?

Comment (gildub) [reply pkg/utils/test/load_test_data.go]: Yes definitely, that was my plan in following patch/PR to push it up.

Comment (arapov) [reply pkg/utils/test/load_test_data.go]: Awe! :)

**Pull request merged**